load_initial_data.sql

```diff
@@ -0,0 +1,41 @@
+DROP TABLE IF EXISTS tUser;
+CREATE TABLE tUser (
```

Comment:
Why table `tUser` does not have password column but your model has it?

```go
type User struct {
	Id       string `json:"user_id"`  // user id is actually the keycloak user id
	Username string `json:"username"` // this should be the keycloak username
	Password string `json:"password"`
	Email    string `json:"email"`
}
```

Reply:
I did it this way as I had in mind an integration with keycloak. So for user creation I would need the password and via Keycloak REST Api I would create a user there, but for the user data required in our DB password field is not needed.
In our DB we will need a user entry in order to associate this with the Keycloak entity.